spec/unit/mutant/matcher/compiler_spec.rb

```diff
@@ -31,32 +31,47 @@
     end
 
     context 'on config with match expression' do
-      context 'and no filter' do
+      let(:expected_predicate) do
+        Morpher::Evaluator::Predicate::Boolean::And.new(
+          [
+            Morpher::Evaluator::Predicate::Negation.new(
+              Morpher::Evaluator::Predicate::Boolean::Or.new(ignore_expression_predicates)
+            ),
+            Morpher::Evaluator::Predicate::Boolean::And.new(subject_filter_predicates)
+          ]
+        )
+      end
+
+      let(:expected_positive_matcher)    { Mutant::Matcher::Chain.new([matcher_a]) }
+      let(:attributes)                   { { match_expressions: [expression_a] }   }
+      let(:ignore_expression_predicates) { [] }
```

Comment:
Alignment of the `}` is off (assuming you follow the same alignment rules for `let` as in other projects, not sure)